docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -781,6 +781,118 @@ To read and write a value of the Bluetooth descriptor:
       - write about setWriteValueRequestCallback() for descriptor
  -->
 ### Responding to read and write value requests from server clients
+
+In order to react and respond to the read and write requsts from a client connected to the server running on the device, the callback functions needs to be set on the characteristics or on the descriptors.
+
+To set a callback function for read or write request on a characteristic or descriptor:
+
+1. Prepare a GATT service containing at least one characteristic or a characteristic with at least one descriptor, that can be read or write:
+   ```
+   var exampleDescriptor = {
+      uuid: "0155",
+      /* Descriptor's permissions */
+      readPermission: true,
+      writePermission: true
+   };
+
+   var exampleCharacteristic = {
+      uuid: "0180",
+      descriptors: [exampleDescriptor],
+      /* Characteristic's properties */
+      isReadable: true,
+      isWritable: true,
+      /* Characteristic's permissions */
+      readPermission: true,
+      writePermission: true
+   };
+
+   var gattService = {
+      serviceUuid: "0955",
+      isPrimary: false,
```

Comment:
Please, change this to `true` - according to the BT spec:

> A secondary service is a service that is only intended to be included from a primary service or another secondary service or other higher layer specification.

As there's no master service for gattService, it probably can't even be registered.